library/pk_wrap.c

```diff
@@ -105,7 +259,85 @@ static int rsa_verify_wrap( void *ctx, mbedtls_md_type_t md_alg,
 
     return( 0 );
 }
+#endif
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
@joerchan : Vestavind do not permit the toup tag unfortunately.

Reply:
https://github.com/nrfconnect/sdk-mbedtls/pull/15/commits/75b92500d9d2bd83066f7633b56c2bd03322265f